package-lock.json

```diff
@@ -995,7 +995,7 @@
         "kmlmp": "build/src/kmlmp.js"
       },
       "devDependencies": {
-        "@sentry/cli": "^2.19.4",
+        "@sentry/cli": "^2.31.0",
```

Comment:
How come I don't see this line in `./package.json`?

Reply:
It's a dev dependency for kmn-kmc and web only; the top-level package-lock.json covers all the subprojects.